apps/betterangels/src/app/(private-screens)/add-note/DateAndTime/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { useMutation } from '@apollo/client';
```

Comment:
Hi @Davit-BetterAngels ,

While I am still waiting for the review on the hifi and attaching the mockup after that, I found 2 things: 

1. Validation message: The message should be visible when the text field is empty. 
2. Placeholder text: It should say "Enter purpose" at least.

Now, those 2 things are not mentioned in the AC so let me know if we need to create a ticket for this.

<img src="https://github.com/user-attachments/assets/0b9b646f-6bc7-4dc9-849a-21c601e771b9" width="360" />